src/content/advanced/storage/azure-pvc-encryption/index.md

```diff
@@ -0,0 +1,78 @@
+---
+linkTitle: Persistent Volumes data encryption with Azure Key Vault
+title: Encrypting Persistent Volumes using Azure Key Vault
+description: Tutorial on how to encrypt dynamically provisioned Persistent Volumes on Azure clusters using Azure Key Vault feature.
+weight: 40
+menu:
+  main:
+    parent: advanced-storage
+user_questions:
+  - How do I encrypt data on Persistent Volumes?
+  - How do I secure data on Persistent Volumes?
+aliases:
+  - /guides/encrypting-persistent-volumes-on-azure-with-azure-key-vault/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-celestial
+---
+
+# Using Persistent Volumes on Azure
+
+If your cluster is running in Azure infrastructure, most common way to provision Persistent Volumes is by using the Azure Disks.
+
+On GS clusters default Storage Classes include already definitions for Azure Disks as well as Azure Files. Those disks however are not encrypted with Customer-Managed keys but only with Platform-Managed keys.
+In order to increase the security of your data persisting on PVs you can facilitate [Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/) service as an encryption key provider.
+
+## Azure Key Vault setup
+
+Prerequisite for next steps is to have a deployed Key Vault instance either in your clusters Resource Group or in your account. Please follow the [official docs](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) to get started.
+
+## Create first encryption key
+
+After successful deployment we can move to creating encryption key that will be used to encrypt data on your PVs.
+
+Add a new key with key type, RSA key size, activation or expiration date of your choosing in the Key Vault directory.
+
+When creation is finished, the Key Identifier should look similar to:
+```https://name-of-your-key-vault-instance.vault.azure.net/keys/test-vault-enc-key/xxxaaadddwww222111dddaaa23456789```
```

Comment:
For an inline code block I think you should use single backticks. If you want to keep it as a code block, then the triple backticks should be on their own lines or the rendering can get confused